docker-compose.yml

```diff
@@ -7,6 +7,21 @@ services:
     ports:
       - "5432:5432"
 
+  app-frontend:
+    image: node:4.4.7-wheezy
```

Comment:
There might have been at one point, but it seems to work with slim just fine so I'll switch.